hs_file_types/models/model_program.py

```diff
@@ -0,0 +1,500 @@
+                                                   id="file_operating_systems", maxlength="250",
+                                                   name="operating_systems", type="text")
+                                dom_tags.label('Programming Languages', fr="file_programming_languages",
+                                               cls="control-label")
+                                with dom_tags.div(cls="controls"):
+                                    programming_languages = self.programming_languages_as_string
+                                    dom_tags.input(value=programming_languages,
+                                                   cls="form-control input-sm textinput textInput",
+                                                   id="file_programming_languages", maxlength="250",
+                                                   name="programming_languages", type="text")
+
+                                with dom_tags.div(cls="controls"):
+                                    json_schema = self.logical_file.metadata_schema_json
+                                    if json_schema:
+                                        json_schema = json.dumps(json_schema, indent=4)
+                                    else:
+                                        json_schema = ''
+
+                                    with dom_tags.fieldset(cls="fieldset-border"):
+                                        with dom_tags.legend("Model Instance Metadata Schema", cls="legend-border",
+                                                             style="font-size: 14px; font-weight:bold;"):
+                                            dom_tags.span(cls="glyphicon glyphicon-info-sign text-muted",
+                                                          data_toggle="tooltip",
+                                                          data_placement="auto",
+                                                          data_original_title="Upload/select a JSON file containing "
+                                                                              "the schema. The content of the file "
+                                                                              "will be shown below when you "
+                                                                              "save metadata.")
+
+                                        # give an option to upload/select a json file for the metadata schema
+                                        with dom_tags.div(cls="form-group"):
+                                            with dom_tags.select(cls="form-control", name='mi_json_schema_template',
+                                                                 style="margin-top:10px;"):
+                                                dom_tags.option("Select a schema", value="")
+                                                template_path = "hs_file_types/model_meta_schema_templates/*.json"
```

Comment:
I believe it is better for this template_path to go to settings.py rather than hardcoded in the code so that it can be changed/configurable if needed.

Reply:
Made the change as suggested.